apps/site/assets/ts/projects/components/FeaturedProjectsList.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { ReactElement } from "react";
+import { SimpleProject as Project } from "./__projects";
+import FeaturedProjectsRow from "./FeaturedProjectsRow";
+
+interface Props {
+  projects: Project[];
+  placeholderImageUrl: string;
+}
+
+const FeaturedProjectsList = ({
+  projects,
+  placeholderImageUrl
+}: Props): ReactElement<HTMLElement> => {
+  const doGroupPairwise = (
```

Comment:
do `doGroupPairwise` and `groupPairWise` need to be defined inside `FeatureProjectList`?  If not, might be better to put them above rather than inside